src/js/shield_defs.js

```diff
@@ -775,11 +775,14 @@ export function loadShields() {
     ["BUS"],
     Color.shields.green
   );
-  shields["US:CA:CR"] = pentagonUpShield(
-    3,
-    15,
-    Color.shields.blue,
-    Color.shields.yellow
+  ["CR", "Sierra"].forEach(
```

Comment:
I take it Sierra County is treated specially because these county routes don’t follow the statewide zone-based numbering scheme. 👍

Reply:
State-coordinated county roads (`US:CA:CR`) coexist with county-assigned routes here, but Sierra County happened to choose the same shield design as Caltrans.

A similar situation appears in Michigan, with Benzie, Gogebic, Kalkaska, Montcalm and Roscommon Counties adopting blue pentagon shields identical to the state-coordinated `US:MI:CR` shields.